.github/workflows/ci.yml

```diff
@@ -32,10 +32,10 @@ jobs:
             os: macos-latest
             build-name: isograph_cli
             artifact-name: isograph_cli-macos-arm64
-          # - target: x86_64-pc-windows-msvc
-          #   os: windows-latest
-          #   build-name: isograph_cli.exe
-          #   artifact-name: isograph_cli-bin-win-x64
+          - target: x86_64-pc-windows-msvc
+            os: windows-latest
+            build-name: isograph_cli.exe
+            artifact-name: isograph_cli-bin-win-x64
```

Comment:
Could you please add a field here (e.g. longpaths: true), as well as the same field for the other OS's, and not have a special case for windows below?

Reply:
added it similar to how it's done for the cargo **cross** compilation, it's not needed for UNIX based systems(linux, macos etc)